Bld/build.sh

```diff
@@ -27,7 +27,7 @@ if [ $? -eq 0 ]; then
     echo -e "${GREEN} ----------------------------------${NOCOLOR}"
     echo -e "${GREEN} ----------------------------------${NOCOLOR}"
     echo -e "${GREEN} Shortcuts:: (add the following lines (aliases) to your bash_profile) ${NOCOLOR}"
-    echo -e "${ORANGE} alias pcl='dotnet ${PWD}/Bld/Drops/Release/Binaries/net6.0/p.dll'${NOCOLOR}"
```

Comment:
lets rename it to `pl` to refer to local version. That is different from the one installed using `dotnet tool`

Reply:
Resolved in 088a3fad5d48b6b694a515ee95746d274f2e52a1